contracts/core/RevenuePath.sol

```diff
@@ -603,19 +592,21 @@ contract RevenuePath is Ownable, Initializable, ReentrancyGuard {
             revert InsufficientWithdrawalBalance();
         }
 
+        uint256 thisRelease;
```

Comment:
What's the purpose of having a local, do you want to emit this? Or is it for readability?

Reply:
this local saves a write to storage on old lines 608 & 614 where `totalReleased` was incremented. Now we only increment `totalReleased` once on new line 608